build.gradle

```diff
@@ -540,7 +540,7 @@ dependencies {
     runtimeOnly 'io.jsonwebtoken:jjwt-jackson:0.10.8'
     runtimeOnly 'com.google.guava:failureaccess:1.0.1'
     runtimeOnly 'org.apache.commons:commons-text:1.10.0'
-    runtimeOnly 'org.glassfish.jaxb:jaxb-runtime:2.3.4'
+    runtimeOnly 'org.glassfish.jaxb:jaxb-runtime:4.0.3'
```

Comment:
@scrawfor99 sorry, I am late, but 2.x is `javax.*` based whereas 3.x+ is `jakarta.*` based, I think this is not compatible change

Reply:
@reta Does this need to be reverted? I can open a PR to revert the change.

Reply:
@cwperks I think so, we could bump to `2.3.8` though, thank you

https://mvnrepository.com/artifact/org.glassfish.jaxb/jaxb-runtime/2.3.8

Reply:
I opened up a PR to revert the change and bump all jaxb dependencies: https://github.com/opensearch-project/security/pull/2977